samples/apps/cap/py/autogencap/actor_connector.py

```diff
@@ -0,0 +1,43 @@
+# Agent_Sender takes a zmq context, Topic and creates a
+# socket that can publish to that topic. It exposes this functionality
+# using send_msg method
+from abc import ABC, abstractmethod
+
+
+class IActorSender(ABC):
```

Comment:
It would be good to document each method and intention here

Reply:
Yeah good point.  Thanks.